poutyne/framework/experiment.py

```diff
@@ -331,24 +518,58 @@ def load_best_checkpoint(self, *, verbose=False):
         return best_epoch_stats
 
     def load_checkpoint(self, epoch):
+        """
+        Loads the attribute model's weights with the weights at a given checkpoint epoch.
+
+        Args:
+            epoch (int): The checkpoint epoch to load.
+        """
         ckpt_filename = self.best_checkpoint_filename.format(epoch=epoch)
         self.model.load_weights(ckpt_filename)
 
     def load_last_checkpoint(self):
+        """
+        Loads the attribute model's weights with the weights of the last checkpoint.
+        """
         self.model.load_weights(self.model_checkpoint_filename)
 
-    def test(self, test_loader, *, steps=None, load_best_checkpoint=True, load_last_checkpoint=False, seed=42):
-        if seed is not None:
-            # Make training deterministic.
-            random.seed(seed)
-            np.random.seed(seed)
-            torch.manual_seed(seed)
+    def test(self, test_loader, *, steps=None, checkpoint='best', seed=42):
+        """
+        Computes and returns the loss and the metrics of the attribute model on a given test examples
+        loader.
+
+        Args:
+            test_loader: Generator-like object for the test set. See :func:`~Model.fit_generator()` for
+                details on the types of generators supported.
+            steps (int, optional): Number of iterations done on ``generator``.
+                (Defaults the number of steps needed to see the entire dataset)
+            checkpoint (Union[str, int]): Which model checkpoint weights to load for the test evaluation.
+                If 'best', will load the best weights according to ``monitor_metric`` and ``monitor_mode``.
+                If 'last', will load the last model checkpoint. If int, will load the checkpoint of the
+                specified epoch.
+                (Default value = 'best')
+            load_best_checkpoint (bool, optional): Whether or not to load the best checkpoint's weights.
```

Comment:
since checkpoint can take 'best' and 'last' as value, I'm not sure that load_best_checkpoint and load_last_checkpoint are still usefull.

Reply:
Params' documentation was only forgotten. Fixed by f60e4bea9fea620a9e5656e18db6e9b6cdbcdeee.